backend/src/schema/types/type/User.gql

```diff
@@ -3,20 +3,16 @@ type User {
   actorId: String
   name: String
   email: String!
-  slug: String
-  password: String!
+  slug: String!
   avatar: String
   coverImg: String
-  avatarUpload: Upload
   deleted: Boolean
   disabled: Boolean
   disabledBy: User @relation(name: "DISABLED", direction: "IN")
   role: UserGroup
   publicKey: String
-  privateKey: String
-  
```

Comment:
> [<img alt="mattwr18" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/mattwr18) **Authored by [mattwr18](https://github.com/mattwr18)**
_<time datetime="2019-07-03T18:41:19Z" title="Wednesday, July 3rd 2019, 8:41:19 pm +02:00">Jul 3, 2019</time>_

---
> **Outdated (history rewrite)** - original diff
---
```diff
@@ -3,20 +3,16 @@ type User {
   actorId: String
   name: String
   email: String!
-  slug: String
-  password: String!
+  slug: String!
   avatar: String
   coverImg: String
-  avatarUpload: Upload
   deleted: Boolean
   disabled: Boolean
   disabledBy: User @relation(name: "DISABLED", direction: "IN")
   role: UserGroup
   publicKey: String
-  privateKey: String
-  
```


this is why you removed the tests that @Tirokk added to check that we can't ask for a user's `privateKey`
meaning if it is removed from the type, we are unable to ask for it.

Reply:
> [<img alt="roschaefer" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/roschaefer) **Authored by [roschaefer](https://github.com/roschaefer)**
_<time datetime="2019-07-03T20:17:38Z" title="Wednesday, July 3rd 2019, 10:17:38 pm +02:00">Jul 3, 2019</time>_

---


Exactly, it is absolutely silly to declare it on the `User` type and then to make sure it's never exposed. Only because of `neo4j-graphql-js` it was there.

Reply:
> [<img alt="mattwr18" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/mattwr18) **Authored by [mattwr18](https://github.com/mattwr18)**
_<time datetime="2019-07-03T21:31:38Z" title="Wednesday, July 3rd 2019, 11:31:38 pm +02:00">Jul 3, 2019</time>_

---


great! another really nice side effect to not using it in this use case :smile_cat: